adobe_vipm/adobe/utils.py

```diff
@@ -45,28 +45,6 @@ def join_phone_number(phone: dict) -> str:
     return f"{phone['prefix']}{phone['number']}" if phone else ""
 
 
-def get_3yc_commitment(customer_or_transfer_preview):
```

Comment:
Didn't get why do you prefer to remove it from adobe/utils.py and leave it in utils.py

Because `get_3yc_commitment_request` function is in adobe/utils.py also

Could you check in your next PRs?